gocwebtemplate-sample-spring/src/main/resources/samples/FeedbackAndShareThisPageSample.html

```diff
@@ -20,24 +20,38 @@ <h2>Feedback Link</h2>
-		    <li>or set the property programmatically via the <code class="wb-prettify">setFeedbackUrl</code> method and optionally the <code class="wb-prettify">setFeedbackUrlFr</code> method in your custom bean class.</li>
+		    <li>or set the property programmatically by creating a FeedbackLink object and enabling it via the <code class="wb-prettify">setEnabled</code> method in your custom bean class.</li>
+		    <li>Your page MUST have the following metadata defined for the feedback tool to be enabled: <code>&lt;meta name="dcterms.creator" content="[Department name / Nom du d�partement]"&gt;</code></li>
+		    <li>The values for Theme, Section and Department Name should be the SAME for English and French.</li>
+		    <li>If you wish to add contact information, set the key <code class="wb-prettify">"goc.webtemplate.feedbackurl"</code> and optionally <code class="wb-prettify">"goc.webtemplate.feedbackurl_fr"</code> in the cdn.properties file which will be the url for the contact link.</li>
+		    <li>or set the property programmatically via the <code class="wb-prettify">setUrl</code> method and optionally the <code class="wb-prettify">setUrlFr</code> method in your custom bean class.</li>
+		    <li>If you wish to add contact information, set the key <code class="wb-prettify">"goc.webtemplate.feedbacktext"</code> and optionally <code class="wb-prettify">"goc.webtemplate.feedbacktext"</code> in the cdn.properties file which will be the name of the contact.</li>
+		    <li>or set the property programmatically via the <code class="wb-prettify">setText</code> method and optionally the <code class="wb-prettify">setTextFr</code> method in your custom bean class.</li>
 		    <li>If no french URL is specified, the english one wil be used for both languages.</li>
+		    <li>If you wish to add the theme, set the key <code class="wb-prettify">"goc.webtemplate.feedbacktheme"</code> in the cdn.properties file.</li>
+		    <li>or set the property programmatically via the <code class="wb-prettify">setTheme</code> method in your custom bean class.</li>
+		    <li>If you wish to add the section, set the key <code class="wb-prettify">"goc.webtemplate.feedbacksection"</code> in the cdn.properties file.</li>
+		    <li>or set the property programmatically via the <code class="wb-prettify">setSection</code> method in your custom bean class.</li>
 		</ul>
 		<div class="wb-prettify all-pre lang-vb linenums">
 		    <pre>
 @Override
 public void onWebTemplateInitialize() {
 //...
-    this.setShowFeedbackLink(true);
-    this.setFeedbackUrl("http://www.google.ca");
-    this.setFeedbackUrlFr("http://www.google.ca/?hl=fr");
+    FeedbackLink feedbackLink = new FeedbackLink();
```

Comment:
(Both sample) sample code doesn't match sample bean

Reply:
done

Reply:
eh.... french character still broken :)

Reply:
I think it should be good now. 

It's really weird. I had made this change but somehow GIT isn't picking it up. For now, I just used the inline editor.